utils/globals.py

```diff
@@ -30,7 +33,63 @@ def set_lab(lab):
 def get_lab():
     if _LAB is None:
         modlog.error('get_lab called before set_lab')
-        #modlog.error('An unexpected error occurd')re
         sys.exit(1)
     return _LAB
 
+
+def set_sampler(sampler_name, version):
+    ''' sets the name and version of the sampler as the sampler UID
+
+    :param sampler_name: name of the method used for samplng
+    :param version: version of the method used for sampling reported
+                    by the sampling function
+    :returns: **no returns, call func get_sampler_uid
+    '''
+    global _SAMPLER, _SAMPLER_has_been_set, _SAMPLER_UID
+
+    _SAMPLER = sampler_name
+    _SAMPLERVERSION = version
+    _SAMPLER_UID = 'escalate_' + str(sampler_name) + '_' + str(version)
+
+    _SAMPLER_has_been_set = True
+
+
+def get_sampler_uid():
+    '''returns the sampler used for automated sampling
+    '''
+    global _SAMPLER_UID
+    if _SAMPLER is None:
+        modlog.error('get_sampler_uid called before set_sampler')
+        sys.exit(1)
+    return _SAMPLER_UID
+
+
+def get_sampler_author():
+    import devconfig
+    sampler_author = 'escalate_' + str(devconfig.RoboVersion)
+    return(sampler_author)
+
+
+def get_user_author_name():
+    message = ("Enter the name you would like to appear for manual submissions (e.g., Ian Pendleton):")
+    user_input = input(message)
+
+    with open('./capture/user_cli_variables.py', 'w') as fout:
+        fout.write("# This file acts as a local cache to the user's preferred author name\n")
+        fout.write("user_author_name = '%s'\n" % user_input)
+    # import the repo_path from the python file cache we just created
+    from capture.user_cli_variables import user_author_name
+    return user_author_name
+
+
+def get_manualruns_uid():
+    manual_run_uid = 'escalate_ManualSpecification_1'
+    return manual_run_uid
```

Comment:
Why isn't this 
`manual_run_uid = f"{get_manualruns_author()}_ManualSpecification_1"`

since the user is explicitly not using ESCALATE and instead doing the 'sampling' themselves.

Reply:
Author is a different column in the file.  We don't want the author's name as part of the model UID.  Manual runs are a type of run that should be bundled together (i.e. they have a property that is associated with all of them that we want to ensure correspondence between).  If you don't want manual runs in your final dataset you can just filter by the model name and remove.   If you want to analyze all of the manual runs, but sort by the author you can also do that.  

This keeps with the conventions built here: https://docs.google.com/spreadsheets/d/1qCvqF1Ix-c7-a3tUr9ZBsaKUJGUB9qxcFfNnqCD3M24/edit#gid=0